Dockerfile

```diff
@@ -67,7 +67,7 @@ RUN set -ex; \
     imagemagick \
     jq \
     libpng-dev \
-    mysql-client \
+    mariadb-client \
```

Comment:
This is not related to nightwatch, just a change in base image, mysql client is replaced by mariadb client.

Reply:
can we know why?

Reply:
I don't know why, base distribution probably change that regarding the license. It's for the same reason that RMF website deploy failed this week.